src/database/DataTypes/Transaction.js

```diff
@@ -597,6 +597,36 @@ export class Transaction extends Realm.Object {
   get isRemoteOrder() {
     return !this.linkedRequisition;
   }
+
+  toJSON() {
+    return {
+      id: this.id,
+      serialNumber: this.serialNumber,
+      otherParty: this.otherParty,
+      comment: this.comment,
+      entryDate: this.entryDate?.getTime(),
+      type: this.type,
+      status: this.status,
+      confirmDate: this.confirmDate?.getTime(),
+      // enteredBy: { type: 'User', optional: true },
```

Comment:
was just checking that it is ok to remove all of these fields and have them undefined?
I think so.. wasn't sure.